.rubocop.yml

```diff
@@ -0,0 +1,21 @@
+Metrics/LineLength:
+  Max: 120
```

Comment:
Google Objc style-guide limits each lines to max 100 character (with some exceptions). Not sure if Ruby sets the standard to 120. If not, do you feel strongly about 120 or can we change it to 100?